Resources/Locale/en-US/_NF/advertisements/vending/vandalizedclothesmate.ftl

```diff
@@ -0,0 +1,7 @@
+﻿advertisement-vandalizedclothes-1 = I don't dig your looks, choom! Get preem threads here!
+advertisement-vandalizedclothes-2 = Don'tcha wanna look totally preem?
+advertisement-vandalizedclothes-3 = D'you wanna look preem or what?! Shlep over here!
+advertisement-vandalizedclothes-4 = Your new look will flatline your input!
+advertisement-vandalizedclothes-5 = Now with added composite armor plates!
+advertisement-vandalizedclothes-6 = Preem threads! Nova getups! Check 'em out, choom!
+advertisement-vandalizedclothes-7 = Nova outfit you have goin' there! For A GONK!
```

Comment:
Reminds me - should the accent have your->ya?

Suggesting preem->nova once because it's used a lot here.
```suggestion
﻿advertisement-vandalizedclothes-1 = I don't dig ya looks, choom! Get preem threads here!
advertisement-vandalizedclothes-2 = Don'tcha wanna look totally nova?
advertisement-vandalizedclothes-3 = D'you wanna look preem or what?! Shlep over here!
advertisement-vandalizedclothes-4 = Your new look will flatline your input!
advertisement-vandalizedclothes-5 = Now with added composite armor plates!
advertisement-vandalizedclothes-6 = Preem threads! Nova getups! Check 'em out, choom!
advertisement-vandalizedclothes-7 = Nova outfit you have goin' there! For A GONK!
```